chart/templates/webserver/webserver-ingress.yaml

```diff
@@ -65,6 +65,7 @@ spec:
         {{- .Values.ingress.web.hosts | default (list .Values.ingress.web.host) | toYaml | nindent 8 }}
       secretName: {{ .Values.ingress.web.tls.secretName }}
   {{- end }}
+  {{- $fullname := (include "airflow.fullname" .) }}
```

Comment:
nit: should we just use this fullname variable for the ingress name as well?

Reply:
good one - addressed in next commit https://github.com/apache/airflow/pull/31066/commits/5fb9f8e05b9c55516ba6a161775892dc396a6335